Tribler/Core/CacheDB/SqliteCacheDBHandler.py

```diff
@@ -1078,31 +1077,8 @@ def freeSpace(self, torrents2del):
         # sql_insert =  "insert into Torrent (torrent_id, infohash, relevance) values (?,?,?)"
         # self._db.executemany(sql_insert, torrent_id_infohashes)
 
-        torrent_dir = self.session.get_torrent_collecting_dir()
         deleted = 0  # deleted any file?
         insert_files = []
```

Comment:
I would create an issue for this, as without this limit some users will start collecting ridiculous amounts of torrents.
I think @synctext had more than 300.000 at some point.